### DISCUSSION TITLE
[feature request] Missing Infill tuning - multiply infill lines, walls, angles (CURA features)

### DISCUSSION THREAD

**Discussion by arciarc** [OrcaSlicer/OrcaSlicer#465] [Ideas]
I enjoy teh Bambu Studio Fork "SoftFever", thanks for the incredible work!

Used to Cura I miss some very usefull functions such as:

    Infill Lines Multiplier
    Infill Walls Multiplier
    Infill Angles (changing every layer, according to a set of chosen angles)

Maybe someone of the programming team can add these features, they don´t appear in Bambu Studio and in SoftFever.
Thank you!

See screenshots in Bambulab-Forum with the same headline.
Keep it going!

**Comment by MakisJacob** [comment]
> I enjoy teh Bambu Studio Fork "SoftFever", thanks for the incredible work!
> 
> Used to Cura I miss some very usefull functions such as:
> 
> ```
> Infill Lines Multiplier
> Infill Walls Multiplier
> Infill Angles (changing every layer, according to a set of chosen angles)
> ```
> 
> Maybe someone of the programming team can add these features, they don´t appear in Bambu Studio and in SoftFever. Thank you!
> 
> See screenshots in Bambulab-Forum with the same headline. Keep it going!

Same issue here!! Did you find out a way to solve this? I really like OrcaSlicer too, but It's getting annoying when you can not perform such simple things when you have the ability to do more complex operations.
